api/src/main/java/org/apache/iceberg/Schema.java

```diff
@@ -355,9 +397,15 @@ private Schema internalSelect(Collection<String> names, boolean caseSensitive) {
 
   @Override
   public String toString() {
-    return String.format("table {\n%s\n}",
-        NEWLINE.join(struct.fields().stream()
-            .map(f -> "  " + f)
-            .collect(Collectors.toList())));
+    StringBuilder sb = new StringBuilder();
+    sb.append("table {\n");
+    sb.append("  fields {\n");
+    sb.append(NEWLINE.join(struct.fields().stream()
+        .map(f -> "    " + f)
+        .collect(Collectors.toList())));
+    sb.append("\n  }\n  identifier fields { ");
+    sb.append(COMMA.join(identifierFieldIds()));
+    sb.append(" }\n}");
+    return sb.toString();
```

Comment:
Minor: I'd prefer a smaller change to the `toString` format. Maybe something like adding `(id)` to identifier fields in the list, rather than adding the `identifier fields { ... }`.